### DISCUSSION TITLE
No hibernate option

### DISCUSSION THREAD

**Discussion by manujchandra** [SpiralLinux/SpiralLinux-project#331] [Q&A]
Hello,

When installing on physical hardware, I choose the option of swap with hibernate (I have 16 GB of RAM, 2 GB is shared with the APU, so usable is 14 GB).

But when I try to shut down XFCE edition, I do not see any option to hibernate. I only see suspend among default options. Usually, when hibernate is enabled, XFCE's shutdown dialog shows hibernate options.

When installing, I saw that a partition of more than 14 GB was created at the end for swap.

Regards,

**Comment by geckolinux** [comment]
Hi there, thanks for the report.

What SpiralLinux ISO version did you use to install?
Could you please post your `/etc/default/grub` file?
Could you please post a screenshot of Gparted showing your disk layout?

Thanks.

**Comment by manujchandra** [comment]
Hi @geckolinux ,

Find the answers to your questions below:

What SpiralLinux ISO version did you use to install?
SpiralLinux_XFCE_12.231008_x86-64.iso

Could you please post your /etc/default/grub file?
Will post be evening when I have access to the computer.

Could you please post a screenshot of Gparted showing your disk layout?
![20231016_192805](https://github.com/SpiralLinux/SpiralLinux-project/assets/15191493/eb2de3c0-8996-400b-85a7-67b76ab6631e)
![20231016_192931](https://github.com/SpiralLinux/SpiralLinux-project/assets/15191493/972b212a-e248-4c07-bffa-a144e84cdbb0)
![20231016_193822](https://github.com/SpiralLinux/SpiralLinux-project/assets/15191493/0eced718-0d94-4541-a9d2-3b33e4b8ca71)
![20231016_193921](https://github.com/SpiralLinux/SpiralLinux-project/assets/15191493/cbc8639c-bf36-4937-95a0-fd3c40c2080c)

**Comment by manujchandra** [comment]
fstab screenshot:
![20231017_132536](https://github.com/SpiralLinux/SpiralLinux-project/assets/15191493/a78f5c6a-98c6-4d1c-98be-53e4a63194dd)

GParted:
![20231017_132629](https://github.com/SpiralLinux/SpiralLinux-project/assets/15191493/6d31c8de-7373-4e6a-a4f2-77f8d1ea79e8)

**Reply by manujchandra** [reply]
Sorry for the ugly screenshots - it's an air-gapped system 🙂
